scalastyle-config.xml

```diff
@@ -9,7 +9,7 @@
  <check level="warning" class="org.scalastyle.file.HeaderMatchesChecker" enabled="true">
   <parameters>
   <parameter name="header"><![CDATA[/*
-  * Copyright 2021 HM Revenue & Customs
+  * Copyright 2022 HM Revenue & Customs
```

Comment:
When I run Scalastyle I get loads of "Header does not match expected text" warnings. This also happens on `main` so it's not due to your change, but could you address it here? I've seen this on a number of repos so far in this story and I've been able to solve it by moving the whole copyright message here one space to the left.